lib/util/assets.ts

```diff
@@ -11,7 +11,7 @@ export const getCurrentPrediction = (
     marketType: { categorical?: string; scalar?: string[] };
     categories: { color: string; name: string; ticker: string }[];
   },
-): string => {
+): [string, number] => {
```

Comment:
Could this return an object so that we can know what these things are via the property names? `[string, number]` is a bit cryptic